aries-site/src/examples/components/tag/data.js

```diff
@@ -0,0 +1,32 @@
+const departments = [
+  'Finance',
+  'Legal',
+  'Engineering',
+  'Design',
+  'Human Resources',
+];
+
+const environments = ['Test', 'Development', 'Production'];
+
+const timezones = ['PST', 'MST', 'EST', 'AET', 'ART', 'BRT', 'GFT'];
+
+const locations = ['NY_US', 'SF_US', 'PAR_FR', 'MD_ES', 'BAR_ES', 'BUD_HU'];
+
+export const tagData = {
+  departments: {
+    displayName: 'Department',
+    options: departments,
```

Comment:
I'd pull these directly into our object since that is how the API would likely return the data.

If we wanted this to be even more realistic, we'd structure this as JSON, then do JSON.parse(data) to format as object, but I'm fine with going straight to object.